WalletWasabi.Packager/usermanuals/macsign.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+cd ~/Desktop/WalletWasabi/WalletWasabi.Packager
+git fetch
+git pull
+dotnet run -- sign
```

Comment:
To avoid issues it might be good to do:

```
git pull || exit "Failed to pull git changes"
```

to exit early (on a git conflict or something)

But then the most safe would be to avoid doing any git operations here at all because one might sign something different than one wants (that would be the worst case scenario I guess), if I understand correctly.

Reply:
ACK

Reply:
![image](https://user-images.githubusercontent.com/9844978/164646756-2056d9c1-8647-4a3f-a0d6-d1db38b6e3a1.png)

Reply:
Then it can be `git pull || exit 1` I guess